documentation/modules/con-considerations-for-data-storage.adoc

```diff
@@ -8,9 +8,19 @@
 
 An efficient data storage infrastructure is essential to the optimal performance of {ProductName}.
 
-{ProductName} requires block storage and is designed to work optimally with cloud-based block storage solutions, including Amazon Elastic Block Store (EBS). The use of file storage (for example, NFS) is not recommended.
+Block storage is a good choice.
```

Comment:
I think we should be stricter here. _Block storage is required. File storage such as NFS does not work with Kafka._